frontend/src/components/common/button/styles.module.scss

```diff
@@ -0,0 +1,55 @@
+.button {
+  box-sizing: border-box;
+  min-width: 42px;
+  padding: 12px 22px;
+  border-width: 1px;
+  border-style: solid;
+  font-family: inherit;
+  font-size: 16px;
+  font-weight: 700;
+  cursor: pointer;
+  transition: all 1s ease;
```

Comment:
```suggestion
  padding: 12px 22px;
  border-width: 1px;
  border-style: solid;
  font-size: 16px;
  font-weight: 700;
  cursor: pointer;
  transition: all 1s ease;
```

after merge it will be by default (because of global styles)